samples/zigbee/light_switch/README.rst

```diff
@@ -25,15 +25,18 @@ The sleepy behavior can be enabled by pressing **Button 3** while the light swit
 Requirements
 ************
 
-* One or more of the following development kits:
+The sample supports the following development kits:
```

Comment:
@ru-fu , @umapraseeda , should this change and changes in lines 33-39 be applied to all samples in NCS?

Reply:
Yes, that's the plan - see https://github.com/nrfconnect/sdk-nrf/pull/2425
But it's impossible to get in all changes at the same time, that's why it's split up in separate PRs. :)